sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/SpecificInternalRow.scala

```diff
@@ -208,10 +208,9 @@ final class SpecificInternalRow(val values: Array[MutableValue]) extends BaseGen
   def this(dataTypes: Seq[DataType]) = {
     // SPARK-32550: use while loop instead of map
```

Comment:
Minor: update comment.

Reply:
Address [9976bf5](https://github.com/apache/spark/pull/29529/commits/9976bf55a6ec7833e31b03433cff09acc696c644) update the comments.